### PR TITLE
Move cos image testing to cos-stable-59-9460-64-0

### PR DESCRIPTION
**Pull request by abgworrall**
This image has a fix needed for ipaliasing.

`$ gcloud compute images list | grep cos-stable` lists the image.

### CODE REVIEW

**Comment by krzyzacy** [comment]
@k8s-bot ok to test

**Comment by krzyzacy** [comment]
/lgtm

wanna merge now?

**Comment by abgworrall** [comment]
Yes please boss !

File: jobs/image/cosbeta.env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 KUBE_NODE_OS_DISTRIBUTION=gci
-KUBE_GCE_NODE_IMAGE=cos-beta-59-9460-57-0
+KUBE_GCE_NODE_IMAGE=cos-stable-59-9460-64-0
```
Comment (wonderfly) [inline jobs/image/cosbeta.env]: Not sure how the file is used but now that a stable image is used, the name `cosbeta` looks contradictory.

**Comment by abgworrall** [comment]
Yeah, we'll get to that soon ;)

**Pull request merged**